source/default/_data/scssVariables.json

```diff
@@ -19,9 +19,9 @@
     "xl": "1200px"
   },
   "scssSpacing": {
-    "s": "5px",
-    "m": "25px",
-    "l": "50px"
+    "s": "rem-calc(5px)",
+    "m": "rem-calc(25px)",
+    "l": "rem-calc(50px)"
```

Comment:
Blaaaarg, the sass2json library is exploding this. We need to toss out these spacing Sass vars anyway (there's already a mixin within Bootstrap providing exactly this that uses all the proper rem values).

I can help remove this sass spacing atom.